### DISCUSSION TITLE
How to know if nofee-ng is working in standalone mode

### DISCUSSION THREAD

**Discussion by jd1900** [nofee-ng/nofee-ng#14] [Q&A] [answered]
I'm running nofee-ng in standalone mode.
But I'm not sure if it's working or not.

I've followed the steps, but the "ip route show" still shows the same information
```
root@pcXX:/opt/nofee-ng# ip route show
default via 192.168.0.1 dev enp34s0 proto dhcp metric 100 
169.254.0.0/16 dev enp34s0 scope link metric 1000 
192.168.0.0/24 dev enp34s0 proto kernel scope link src 192.168.0.16 metric 100
```

It seems to be working fine:
```
root@pcXX:/opt/nofee-ng# systemctl status nofee
● nofee.service - Next Generation NoFee Miner
     Loaded: loaded (/etc/systemd/system/nofee.service; enabled; vendor preset: enabled)
     Active: active (running) since Mon 2021-06-07 12:27:28 CEST; 1min 44s ago
   Main PID: 265655 (nofee_eth)
      Tasks: 1 (limit: 47969)
     Memory: 3.7M
     CGroup: /system.slice/nofee.service
             └─265655 /opt/nofee-ng/nofee_eth

jun 07 12:27:28 pc01 systemd[1]: Started Next Generation NoFee Miner.
jun 07 12:27:28 pc01 nofee_eth[265655]: Found NoFee Wallet: mywallethere
jun 07 12:27:28 pc01 nofee_eth[265655]: Running.
jun 07 12:27:28 pc01 nofee_eth[265655]: !!!!! DO NOT kill this program with 'kill -9' !!!!!
```

The miner works as before.

Should I change anything in the miner in standalone? How can I know if its working?

**Comment by nofee-ng** [comment, marked as answer]
The line "jun 07 12:27:28 pc01 nofee_eth[265655]: Found NoFee Wallet: mywallethere" seems that you have put a wrong refund wallet address in nofee.txt. "mywallethere" should be your refund wallet address which starts with 0x.
If DevFee is found, a line DevFee found will show up in systemctl status nofee. But NOT ALL DevFee found will become valid shares on the pool.

**Reply by jd1900** [reply]
I changed my real address with 'mywallethere'. But there is actually there my real address.
